std_msgs/Header header
geometry_msgs/Point biased_centroid
float64 x_offset
float64 density
bool valid